source/features/dim-bots.tsx

```diff
@@ -12,7 +12,8 @@ export const botSelectors = [
 	'.commit-author[href$="scala-steward"]:first-child',
 
 	/* Issues/PRs */
-	'.opened-by [href*="author%3Aapp%2F"]'
+	'.opened-by [href*="author%3Aapp%2F"]',
+	'.labels [href$="bot"]'
```

Comment:
I think this would make it specific to the `bot` label instead of any label that ends in `bot`
```suggestion
	'.labels [href$="/bot"]'
```